Sources/Sentry/SentryCrashReportConverter.m

```diff
@@ -237,6 +237,9 @@ - (SentryThread *_Nullable)crashedThread {
         SentryDebugMeta *debugMeta = [[SentryDebugMeta alloc] init];
         debugMeta.uuid = sourceImage[@"uuid"];
         debugMeta.type = @"apple";
+        if ([sourceImage[@"image_vmaddr"] integerValue] > 0) {
```

Comment:
Why only when it is `> 0`?

Reply:
To save some payload size, we default on the server to 0 if it's not sent.

Reply:
Can we then add this information as a comment please?